libnavigation-core/src/main/java/com/mapbox/navigation/core/fasterroute/FasterRouteObserver.kt

```diff
@@ -16,11 +16,11 @@ interface FasterRouteObserver {
     /**
      * Invoked whenever a faster route was inspected.
```

Comment:
```suggestion
     * Invoked whenever a faster route is available.
```

Reply:
the callback happens even if the result was full of slower routes, so "whenever faster route is available" doesn't feel accurate 🤔